tests/RulesetTest.php

```diff
@@ -98,6 +98,9 @@ public function __construct( $ruleset, $expected = [] ) {
 			$this->phpcs_bin = realpath( $phpcs_bin );
 		}
 
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
There are quite a few ignore annotations of this sniff and the `system_calls` group of the `WordPress.PHP.DiscouragedPHPFunctions` sniff for this file.

Would it make sense to add an exception to the project `phpcs.xml.dist` file for those two rules for this particular file (and to remove the inline ignore annotations) ?
Alternatively, `// phpcs:disable` comments could be added to the top of this particular file for those error codes.